docs/Makefile.am

```diff
@@ -33,21 +33,20 @@ endif
 
 # EXTRA_DIST breaks with $(abs_builddir) so build it using this variable
 # but distribute it (using the relative file name) in the next variable
-man_MANS = $(abs_builddir)/curl.1
-noinst_man_MANS = $(MK_CA_DOCS)
-dist_man_MANS = $(CURLCONF_DOCS) $(MK_CA_DOCS)
+man_MANS = $(abs_builddir)/curl.1 curl-config.1
+noinst_MANS = mk-ca-bundle.1
```

Comment:
This variable doesn't do anything (you can look at the generated Makefile.in to confirm). The only thing I could find about it in automake source is this comment:

```
## We don't really need this, but we use it in case we ever want to
## support noinst_MANS.
MANS = %MANS%
```